release.nix

```diff
@@ -24,10 +25,11 @@ let
     name = "tezos-release-no-tarball";
     paths = [ "${bundled.binaries}/bin" LICENSE release-notes ];
   };
-  releaseTarball = runCommand "release-tarball" { }
-    "mkdir $out; tar --owner=serokell:1000 --mode='u+rwX' -czhf $out/release.tar.gz -C ${releaseNoTarball} .";
+  tarballName = "binaries-${builtins.stringReplace ["v"] [""] source.ref}-1.tar.gz";
+  binariesTarball = runCommand "binaries-tarball" { }
+    "mkdir $out; tar --owner=serokell:1000 --mode='u+rwX' -czhf $out/${tarballName} -C ${bundled.binaries}/bin .";
```

Comment:
What's the reasoning for this change? Don't we want stuff like LICENSE and release notes included in the tarball?

Reply:
we want a single tarball with all binaries, see old releases

Reply:
Maybe LICENSE is useful, but release notes definitely shouldn't be included